controllers/memberstatus/memberstatus_controller.go

```diff
@@ -93,10 +92,13 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	reqLogger := r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling MemberStatus")
 
-	config, err := cfg.GetConfig(r.Client, request.Namespace)
+	// retrieve the latest config and use it for this reconciliation
+	config, err := memberCfg.GetConfig(r.Client, request.Namespace)
 	if err != nil {
 		return reconcile.Result{}, errs.Wrapf(err, "unable to get MemberOperatorConfig")
 	}
+	r.config = config
```

Comment:
It would be nicer to not store it as a parameter of the `Reconciler` type, but pass it as an argument to all methods/functions. In the same way, as we do with the `MemberStatus` object or with the logger.
After all, it's nothing static but we update it with every reconcile.
 WDYT?

Reply:
Yeah, I agree with you. While working on the host operator changes I decided to pass it as an argument to functions OR if there was only one function that needed config then just call `GetConfig` from that function. I'll update this PR to do the same.

Reply:
Done in 71a0c3e74f849496bfac07f9c3573cef8a80e5e6